shell/platform/darwin/ios/framework/Source/FlutterDartProject.mm

```diff
@@ -207,14 +207,25 @@ + (NSString*)flutterAssetsName:(NSBundle*)bundle {
 }
 
 + (NSString*)lookupKeyForAsset:(NSString*)asset {
-  NSString* flutterAssetsName = [FlutterDartProject flutterAssetsName:[NSBundle mainBundle]];
+  return [self lookupKeyForAsset fromBundle:[NSBundle mainBundle]];
+}
+
++ (NSString*)lookupKeyForAsset:(NSString*)asset fromBundle:(NSBundle*)bundle {
+  NSString* flutterAssetsName = [FlutterDartProject flutterAssetsName:bundle];
   return [NSString stringWithFormat:@"%@/%@", flutterAssetsName, asset];
 }
 
 + (NSString*)lookupKeyForAsset:(NSString*)asset fromPackage:(NSString*)package {
   return [self lookupKeyForAsset:[NSString stringWithFormat:@"packages/%@/%@", package, asset]];
```

Comment:
I'd prefer leveraging `lookupKeyForAsset:fromPackage:fromBundle:` here.  Duplicating `[NSBundle mainBundle]` seems less error-prone than duplicating `[NSString stringWithFormat:@"packages/%@/%@", package, asset]`.

Otherwise looks good to me.

Reply:
Updated this and another spot where there was duplicated code.